l10n_de_tax_statement/views/l10n_de_tax_statement_view.xml

```diff
@@ -104,6 +104,11 @@
                         </page>
                     </notebook>
                 </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" widget="mail_followers"/>
+                    <field name="message_ids" widget="mail_thread"/>
+                    <field name="activity_ids" widget="mail_activity"></field>
```

Comment:
It will be better, if you replace this line with this ;)
`<field name="activity_ids" widget="mail_activity"/>`